wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -237,11 +246,11 @@
       },
       {
         "package": "SwiftProtobuf",
-        "repositoryURL": "https://github.com/apple/swift-protobuf.git",
+        "repositoryURL": "https://github.com/wireapp/swift-protobuf",
         "state": {
           "branch": null,
-          "revision": "e17d61f26df0f0e06f58f6977ba05a097a720106",
```

Comment:
This is a little interesting. This is from the workspace. This was previously pointing to the version used by a `DataDog` in `WireAnalytics` which I don't believe we are explicitly using. However now it points to the version we are explicitly importing which was the same version defined in the Cartfile. I _think_ all is still OK.